Formula/g/gollama.rb

```diff
@@ -0,0 +1,23 @@
+class Gollama < Formula
+  desc "Go manage your Ollama models"
+  homepage "https://smcleod.net"
+  url "https://github.com/sammcj/gollama/archive/refs/tags/vv1.20.5.tar.gz"
```

Comment:
This doesn't seem to be released yet

Reply:
That indeed seems to be the case - I have fixed it now, thank you!